### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by Seth1771** [ImpactDevelopment/ImpactIssues#2556]
**Describe the bug**
Generic "Game Crashed unexpected error" pops up when trying to launch

**To Reproduce**
1.) Try to launch Impact
**Expected behavior**
Hopefully use Impact

- Impact Version: 
- Using Optifine? 
- Using LiteLoader? 
- Enabled mods: 
- Operating System: 

-----Just updated to the latest Java, JAVA is not the issue here, here's my latest log -----
[latest.log](https://github.com/ImpactDevelopment/ImpactIssues/files/5011556/latest.log)

**Comment by AbsoluteBananas** [comment]
i'm having the same problem, someone at impact development told me to download jdk 8, mabye that could help for you?
(it didnt help for me)

**Comment by Seth1771** [comment]
> i'm having the same problem, someone at impact development told me to download jdk 8, mabye that could help for you?
> (it didnt help for me)

yeah I already have the latest version for Java, I don't think java is the issue here

**Comment by AbsoluteBananas** [comment]
same

**Comment by AbsoluteBananas** [comment]
when dominika comes online ill close my query 

and ill kinda come onboard with you

as we have same problem

**Comment by Seth1771** [comment]
> same

I went to the Discord for help and was told to post here about it, so here's to hoping lmao

**Comment by AbsoluteBananas** [comment]
lmao

**Comment by AbsoluteBananas** [comment]
whats the discord?

**Comment by ScanmanTM** [comment]
> Just updated to the latest Java, JAVA
what does this even mean?? What java version.

**Comment by AbsoluteBananas** [comment]
jdk 8

**Comment by dominikaaaa** [comment]
bruh what

I mean to me this doesn't look like impacts fault, considering how little information the log has

**Comment by Seth1771** [comment]
> bruh what
> 
> I mean to me this doesn't look like impacts fault, considering how little information the log has

so what would this be then? I don’t see how it could be anything but. Impact 4.9 for 1.12.2 works just fine, but not 4.9 for 1.15.2, so it has to be Impact.

**Comment by AbsoluteBananas** [comment]
yes, mine works for everything else but impact

**Comment by Seth1771** [comment]
> > Just updated to the latest Java, JAVA
> 
> what does this even mean?? What java version.

"Just updated to the latest Java, JAVA is not the issue here" it means what it says

**Comment by Seth1771** [comment]
*Still unresolved 8/3/20

**Comment by dominikaaaa** [comment]
ok

**Comment by dominikaaaa** [comment]
- dominika 2020/08/03 13:50

**Comment by Seth1771** [comment]
*Still unresolved 8/5/20 | 8/6/20

**Comment by dominikaaaa** [comment]
ok

**Comment by dominikaaaa** [comment]
- dominika 2020/08/05 22:05

**Comment by Darkstalker** [comment]
I am having the same problem.

**Comment by Darkstalker** [comment]
Which unfortunately means i can't run my anarchy server.

**Comment by dominikaaaa** [comment]
Thank you for your very useful comments

**Comment by Seth1771** [comment]
Not spam- Multiple people having the same issue, unresolved with no assistance or inquiries

**Comment by Seth1771** [comment]
> Thank you for your very useful comments

thank you for your input and delegations on these matters

**Comment by dominikaaaa** [comment]
Putting "I also have the same issue" with no further information doesn't help anybody and is spam. This is just another one of the 19485828949 crash issues which are related to things specific about your setup. If you searched this GitHub beforehand you'd see that there's hundreds of closed issues about crashes caused by user error.

**Comment by Darkstalker** [comment]
oh my bad i thought this was just a place to let who ever is inchage that i had a problem with impact

**Comment by Seth1771** [comment]
> oh my bad i thought this was just a place to let who ever is inchage that i had a problem with impact

that is exactly what it is, it would be appreciated for someone to look into this too, seeing as how many players are having this issue

**Comment by Seth1771** [comment]
> Putting "I also have the same issue" with no further information doesn't help anybody and is spam. This is just another one of the 19485828949 crash issues which are related to things specific about your setup. If you searched this GitHub beforehand you'd see that there's hundreds of closed issues about crashes caused by user error.

yeah i’ve looked, can’t seem to find one though. Mind linking me instead of marking everything as spam? :)

**Comment by dominikaaaa** [comment]
##### #2377

#### #2294

### #2280

## #2085

# #1677

# #2232

# #2196
